plugins/cpu/setup.py

```diff
@@ -1,3 +1,8 @@
+# add `build_util` to import path
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
```

Comment:
what's this line for?

Reply:
See the comment and PR description. `setuptools` doesn't include the local project directory in the `PYTHONPATH` by default when using `pip install`/`pip wheel`. Slightly re-ordered lines.